enterprise/glassfish.tooling/src/org/netbeans/modules/glassfish/tooling/server/config/GlassFishV6_1_0.xml

```diff
@@ -23,9 +23,10 @@
     <tools lib="lib">
         <asadmin jar="client/appserver-cli.jar"/>
     </tools>
-    <java version="1.8">
-        <platform version="1.7"/>
-        <platform version="1.8"/>
+    <java version="11">
+        <platform version="1.8"/> 
```

Comment:
For me running glassfish on 6.1 on 8 failed. Are you sure 8 is still supported?

Reply:
Glassfish 6.1 and 6.2 correspond with Jakarta 9.1 which runs on JDK 11, I understand that it should run on JDK 11 or higher. 
https://github.com/eclipse-ee4j/glassfish/releases

Reply:
I saw conflicting info in different places. It basically says that Jdk11 Is supported, but I do not see it mandatory. Also JakartaEE 10 will require 11, as of it was a new requirement. @juneau001 PR #2902 did not require 11. On the other hand I was not able to launch with 1.8 from Jdk16, it did not understand rmi opens from Glassfish. Maybe it needs to launch from a Netbeans running on 1.8?